test_data/python_config.yml

```diff
@@ -1,8 +1,5 @@
 ---
 PYTHON_INSTALLATIONS:
-    - PYTHON_ALIAS: 'PY_27'
-      PYTHON_PATH: '/usr/bin/python'
-      PYTHON_PROPERTIES: []
     - PYTHON_ALIAS: 'PY_35'
```

Comment:
I would leave this in. If anything it's there to exercise the ability to install multiple python versions (which we will still need)